src/scripts/page/questionnaire/depistage.js

```diff
@@ -68,14 +101,41 @@ export default function depistage(form, app) {
             )
             app.profil.depistage_type = getRadioValue(form, 'depistage_type')
             app.profil.depistage_resultat = getRadioValue(form, 'depistage_resultat')
+            if (app.profil.depistage_resultat === 'positif') {
+                app.profil.depistage_variante = getRadioValue(
+                    form,
+                    'depistage_variante'
+                )
+            } else {
+                app.profil.depistage_variante = undefined
+            }
         } else {
             app.profil.depistage_start_date = undefined
             app.profil.depistage_type = undefined
             app.profil.depistage_resultat = undefined
+            app.profil.depistage_variante = undefined
         }
 
         app.enregistrerProfilActuel().then(() => {
             app.goToNextPage('depistage')
         })
     })
 }
+
+function showVariante(varianteRadio) {
+    showElement(varianteRadio)
+    varianteRadio.classList.add('required')
+    // eslint-disable-next-line no-extra-semi
+    ;[].forEach.call(varianteRadio.querySelectorAll('input[type=radio'), (elem) => {
```

Comment:
```suggestion
    Array.from(varianteRadio.querySelectorAll('input[type=radio').forEach((elem) => {
```

J'essaye maintenant de passer sur cette variante plus explicite (et on a le polyfill qui passe sous IE) qui ne nécessite pas le commentaire eslint ni le `;` en plus.